style.css

```diff
@@ -322,28 +322,266 @@ textarea:focus {
+    height: 685px;
+    padding-top: 192px;
+    background-image: url("./img/image_dots.svg");
+    background-position: bottom right;
+    background-repeat: no-repeat;
+  }
 
-.get-started p:hover {
-  transform: translate(20px);
+  section.header-image {
+    width: 50%;
+    background-image: url("./img/image-geometry_desktop.svg");
+    height: 877px;
+  }
+
+  h1 {
+    font-size: 110px;
+    line-height: 110px;
+    height: 220px;
+    padding-bottom: 60px;
+  }
+
+  p.introduction {
+    padding-bottom: 60px;
+  }
+
+  button.primary-dark {
+    width: 254px;
+  }
+
+  .mobile-only {
+    display: none;
+  }
+
+  .desktop-only {
+    display: block;
+    margin-right: 50px;
+    font-family: Poppins, sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: right;
+    color: #1a2236;
+    border-bottom: 2px solid #fff;
+  }
+
+  .desktop-only:hover {
+    border-bottom: 2px solid #1a2236;
+  }
+
+  main {
+    margin-top: 200px;
+    grid-template-columns: 11.1% 11.1% 11.1% 11.1% 11.1% 11.1% 11.1% 11.1% 11.1%;
+    grid-template-rows: 558px 80px 190px 80px 190px 275px 270px;
+  }
+
+  main h2 {
+    font-size: 110px;
+    transform: rotate(-90deg);
+    width: 11.1%;
+    height: 50px;
+    grid-column: 2 / 3;
+    position: relative;
+    top: 350px;
+  }
+
+  .item1 {
+    grid-column: 4 / 10;
+  }
+
+  .item2 {
+    grid-column: 1 / 4;
+    grid-row: 3 / 5;
+  }
+
+  .item6 {
+    grid-column: 7 / 10;
+    grid-row: 2 / 4;
+  }
+
+  .item5 {
+    grid-column: 7 / 10;
+    grid-row: 4 / 6;
+  }
+
+  .item4 {
+    grid-column: 6 / 9;
+    grid-row: 7 / 8;
+  }
+
+  .item3 {
+    grid-column: 1 / 6;
+    grid-row: 5 / 7;
+  }
+
+  .spots {
+    background-image: url("./img/image_dots.svg");
+    background-position: top;
+    background-repeat: no-repeat;
+    grid-column: 3 / 4;
+    grid-row: 1 / 2;
+  }
+
+  .background2 {
+    background-image: url("./img/image-geometry_2.png");
+    background-repeat: no-repeat;
+    grid-column: 4 / 7;
+    grid-row: 2 / 5;
+  }
+
+  .background3 {
+    background-image: url("./img/image-geometry_3.png");
+    background-repeat: no-repeat;
+    grid-column: 6 / 10;
+    grid-row: 6 / 7;
+    z-index: -1;
+  }
+
+  .background4 {
+    background-image: url("./img/image-geometry_4.png");
+    background-repeat: no-repeat;
+    grid-column: 1 / 3;
+    grid-row: 7 / 8;
+  }
+
+  .background5 {
+    background-image: url("./img/image-geometry_5.png");
+    background-repeat: no-repeat;
+    background-position: top right;
+    grid-column: 5 / 6;
+    grid-row: 7 / 8;
+  }
+
+  section.about {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 160px;
+  }
+
+  .about-information {
+    width: 40%;
+    margin-left: 10%;
+  }
+
+  .categories {
+    width: 50%;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 50% 50%;
+  }
+
+  li.lists {
+    margin: 0;
+  }
+
```

Comment:
- [ ] As you read in the previous suggestion. The feature project looks great on 992px. However, your table of technologies doesn't. An option could be to reduce the left-right padding of your about section to avoid the issue of your table of techs. (It's not a mandatory option you can find another way to fix it.) 😉 :

![table tech](https://user-images.githubusercontent.com/79876959/139562039-c99eb4c0-98a2-412c-bd37-21b8f9e68909.PNG)